### PR TITLE
outplacement_final_report AFC-1933: fixed study visit GUI

### PR DESCRIPTION
**Pull request by SebastianGieser-Vertel**

### CODE REVIEW

File: outplacement_final_report/i18n/sv_SE.po [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+#. module: outplacement_final_report
+#: model:ir.model.fields,field_description:outplacement_final_report.field_outplacement_goal_step__other_step_name
+#: model:ir.model.fields,field_description:outplacement_final_report.field_outplacement_study_visit__name
+msgid "Name"
+msgstr "Namn"
+
+#. module: outplacement_final_report
+#: model:ir.model.fields,field_description:outplacement_final_report.field_outplacement__obstacle_reason
+msgid "Obstacle Reason"
+msgstr ""
```
Comment (nils-vertel) [inline outplacement_final_report/i18n/sv_SE.po]: Missing translation?

File: outplacement_final_report/i18n/sv_SE.po [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+#: selection:outplacement.goal.step,step_type:0
+msgid "Other"
+msgstr "Annat"
+
+#. module: outplacement_final_report
+#: model:ir.model,name:outplacement_final_report.model_outplacement
+msgid "Outplacement"
+msgstr "Deltagare"
+
+#. module: outplacement_final_report
```
Comment (nils-vertel) [inline outplacement_final_report/i18n/sv_SE.po]: deltagare mål, deltagare mål steg, deltagare studiebesök feels off, please consider rephrasing.

**Review by nils-vertel — CHANGES_REQUESTED** [review]
Some translations seems a bit off to me, check with Tim for second opinion if he approve merge it.

File: outplacement_final_report/i18n/sv_SE.po (hunk 1 above)
Comment (timforsstromAF) [reply outplacement_final_report/i18n/sv_SE.po]: Yes, added that :)

File: outplacement_final_report/i18n/sv_SE.po (hunk 2 above)
Comment (timforsstromAF) [reply outplacement_final_report/i18n/sv_SE.po]: förslag: deltagarens mål, deltagaranes delmål, deltagarens studiebesök?

**Comment by timforsstromAF** [comment]
Change translations as suggested and its ok with me.

**Review by nils-vertel — APPROVED** [review]

**Pull request merged**